README.md

```diff
@@ -15,23 +44,28 @@ system to make contributing to this repository easier.
 
 ### Exercises
 
-All exercises have the following structure:
+All exercises are registered in the file
+- `config.json`: Every exercise has a unique name and a difficulty. The sequence order is also the order in which the exercises are to be solved.
```

Comment:
I would say "order in which the exercises are fetched" or maybe "order in which the exercises are served", but I'm mainly saying that since I solve the exercises out-of-order. If not enough contributors will think about this, then don't mind me.